packages/terra-application-menu-layout/examples/Index.site-page.jsx

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable import/no-extraneous-dependencies, import/no-webpack-loader-syntax, import/first, import/no-unresolved, import/extensions */
+import React from 'react';
```

Comment:
Moved from `terra-framework-site/src/examples/application-menu-layout` & update to use the `IndexPageTemplate`